codegra_fs/constants.py

```diff
@@ -22,8 +22,40 @@
 url_help = """The url to find the api. This defaults to
 'https://codegra.de/api/v1/'. This is most likely
 'https://{UNIVERSITY}.codegra.de/api/v1/' where '{UNIVERSITY}' is your
-university (for example 'uva' for the University of Amsterdam).  This value can
-also be passed as an environment variable 'CGAPI_BASE_URL'"""
+university (for example 'uva' for the University of Amsterdam and 'ut' for the
+University of Twente).  This value can also be passed as an environment
+variable 'CGAPI_BASE_URL'"""
 
 all_submissions_help = """See all submissions not just the latest submissions
 of students."""
+
+cgfs_epilog="""possible environment variables
```

Comment:
I would remove "possible" here or change it to "supported", add a colon to the end, and remove the empty line below this one.